syftlib/src/main/java/org/openmined/syft/processes/Plan.kt

```diff
@@ -0,0 +1,7 @@
+package org.openmined.syft.processes
+
+class Plan(val planId: String) {
+    lateinit var torchScriptLocation: String
```

Comment:
If we pass `torchScriptLocation` as a param we won't need the `lateinit`. 
But I guess we can still consider this calls and `Protocol` to be drafts.

Reply:
this will be defined much later when we actually download the file